src/Microsoft.FeatureManagement/Microsoft.FeatureManagement.csproj

```diff
@@ -3,10 +3,9 @@
 
   <!-- Official Version -->
   <PropertyGroup>
-    <MajorVersion>3</MajorVersion>
-    <MinorVersion>0</MinorVersion>
+    <MajorVersion>2</MajorVersion>
+    <MinorVersion>4</MinorVersion>
```

Comment:
I suppose you will make the version change later.

Reply:
Yes.